docs/source/tutorials/improving_ml_performance.ipynb

```diff
@@ -0,0 +1,3574 @@
+       "              interaction_constraints=None, learning_rate=None, max_bin=None,\n",
+       "              max_cat_threshold=None, max_cat_to_onehot=None,\n",
+       "              max_delta_step=None, max_depth=None, max_leaves=None,\n",
+       "              min_child_weight=None, missing=nan, monotone_constraints=None,\n",
+       "              multi_strategy=None, n_estimators=None, n_jobs=None,\n",
+       "              num_parallel_tree=None, objective='multi:softprob', ...)"
+      ]
+     },
+     "execution_count": 40,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "clean_clf = XGBClassifier(tree_method=\"hist\", enable_categorical=True, random_state=SEED)\n",
+    "clean_clf.fit(train_features, train_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2fe23e39-fe7b-4145-af55-c7fc1f245850",
+   "metadata": {},
+   "source": [
+    "**In a Jupyter environment, please rerun this cell to show the HTML representation or trust the notebook.\n",
+    "On GitHub, the HTML representation is unable to render, please try loading this page with nbviewer.org.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b0f83f8-07e6-4702-a39e-94336268bfef",
+   "metadata": {},
+   "source": [
+    "### Evaluate classification model with clean training on clean test data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "id": "dcaeda51-9b24-4c04-889d-7e63563594fc",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy of model fit to clean training data, measured on clean test data: 77.2%\n"
+     ]
+    }
+   ],
+   "source": [
+    "clean_preds = clean_clf.predict(test_features)\n",
+    "acc_clean = accuracy_score(test_labels, clean_preds)\n",
+    "print(\n",
+    "    f\"Accuracy of model fit to clean training data, measured on clean test data: {round(acc_clean*100,1)}%\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2624388-ad39-4c88-88c3-51a224ad549a",
+   "metadata": {},
+   "source": [
+    "## 8. Hyperparameter Optimization for editing data issues"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96e8e3fe-b15f-41e0-87dd-0efb786f2920",
+   "metadata": {},
+   "source": [
+    "We have made some basic edits to improve test performance, so now we will parameterize each one of these edits (eg. what fraction of each issue to delete) to automatically find the best combination of edits to achieve optimal test performance. \n",
+    "\n",
+    "We will use a basic hyperparameter-tuning approach to optimize over these edit-variants + model re-training on the edited datasets with our objective being test performance.\n",
+    "\n",
+    "In a real-world setting, this would ideally be done on cleaned validation data instead of test data, but we are simplifying the approach for this tutorial.\n",
```

Comment:
```suggestion
    "In a real-world setting, hyperparameter tuning should be evaluated on a third set of data (a validation set) data instead of the test set, but we are simplifying the approach for this tutorial.\n",
```

Honestly though I'd probably just have a validation set (if my understanding of what we're skipping is correct!)